### DISCUSSION TITLE
How can I turn off the Mailchimp signup form?

### DISCUSSION THREAD

**Discussion by bbagwang** [nunocoracao/blowfish#918] [Q&A] [answered]
I keep getting this modal on my site after installing this theme.
I search the code related to this but can't find anything.

Does anyone know how to remove this?

Thanks.

**Comment by nunocoracao** [comment]
Hmm what? How did you install the theme?

**Comment by nunocoracao** [comment, marked as answer]
Hey @bbagwang you didn't install the theme following the docs. Please next time follow the documentation to avoid wasting your time and someone else's. 

You are seeing this behavior because you cloned my personal website and kept files there that change the site. In this case the extend-footer and extend-head 

https://github.com/bbagwang/bbagwang.github.io/blob/main/layouts/partials/extend-footer.html

https://github.com/bbagwang/bbagwang.github.io/blob/main/layouts/partials/extend-head.html

So you are literally loading my mailchimp subscribe popup

**Reply by bbagwang** [reply]
I'll try reinstalling it from scratch.

Thanks for your help and sorry for the inconvenience. 🙏
